gemini/tests/set4/tabs.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
```

Comment:
You could skip that change :)

Reply:
It's not a single change, but a series of two changes, the second of which reverts the first. That's how the date update remained :)